src/india_api/cmd/main.py

```diff
@@ -9,6 +11,16 @@
 
```

Comment:
I think it should be from `india_api.internal.service import get_db_client, server, version`, not `import version`